### ISSUE TITLE
integer expression expected

### ISSUE THREAD

**Issue by MarcHagen** [superjamie/minecraft-init-script#15]
When i run my server and to the info command i get in the middle of the info this line:
/etc/init.d/minecraft line 444: [: : integer expression expected

Running CentOS 6.4 min.
Java JDK 1.7.0_45

**Comment by superjamie** [comment]
That's the HugePages check, could you please show the output of the following:

```
which grep
which awk
cat /proc/meminfo | grep HugePages_Total | awk '{print $2}'
```

You'll need to have both `grep` and `awk` installed. If either of those first two lines return `/usr/bin/which: no <bin> in <path>` you'll need to `yum install grep gawk`.

**Comment by MarcHagen** [comment]
which grep : /bin/grep
which awk : /bin/awk
cat /proc/meminfo | grep HugePages_Total | awk '{print $2}' returns nothing.

Here is my meminfo :
MemTotal:        2097152 kB
MemFree:          832620 kB
Cached:           431928 kB
Active:           249672 kB
Inactive:         261252 kB
Active(anon):      68412 kB
Inactive(anon):    10584 kB
Active(file):     181260 kB
Inactive(file):   250668 kB
Unevictable:           0 kB
Mlocked:               0 kB
SwapTotal:             0 kB
SwapFree:              0 kB
Dirty:                16 kB
Writeback:             0 kB
AnonPages:         78996 kB
Shmem:              2564 kB
Slab:               6496 kB
SReclaimable:       3512 kB
SUnreclaim:         2984 kB

Very sorry for the markup.
Can't do that on the iPhone ..

**Comment by superjamie** [comment]
This should be fixed with SHA: 8c32f6895a2fd0ac9051f4c7c946616858a1802a
